server/api/routers/discussionRouter.js

```diff
@@ -52,6 +52,31 @@ router.get('/:id', (req, res) => {
     });
 });
 
+/*
+GET ALL DISCUSSION ROUTE BY SUBTOPIC_ID - gets all routes by subtopic_id
+@PARAM = {
+    id: !INT
+}
+ROUTE = '/discussions/s/:id
+returns = all discussions with subtopic_id
+TESTS: {
+    1) RETURNS DISCUSSIONS BY SUBTOPIC_ID
+}
+*/
+
+router.get('/s/:id', (req, res) => {
+  const { id } = req.params;
+
+  db('discussion')
```

Comment:
Needs username data joined to returned promise.

Reply:
Added raw SQL statement